modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,108 @@
+= Migrate to the Version 2 of the Mocking Service
+
+The mocking service lets you test endpoints that are defined in an API specification. The advantage to using the mocking service is that you do not need to implement the API before testing. To test an endpoint, the mocking service simulates calls to its methods.
+
+For example, when you are designing an API specification in API Designer, you can simulate a call to an endpoint’s GET method to see whether the response body for the 200 status code is as you want it. As a further example, if you are evaluating whether you want to implement an API specification that is in Exchange, you can simulate a call to an endpoint’s GET method to find out what the response body looks like.
+
+[IMPORTANT]
+====
+After January 31, 2020, version 1 of the mocking service will no longer send responses to requests. If you are using a third-party tool to send requests, the mocking service will return the HTTP status code 301 and provide a link to this document.
+====
+
+== How to Migrate an API-Specification Project to Version 2 of the Mocking Service
+
+If an API-specification project is older than January 10, 2019 and the mocking service was switched on before that date, follow these steps to migrate to version 2 of the mocking service:
+
+. In the notification that appears in the top-right corner after you open the project, click *Upgrade*. The text of the notification reads “A new mocking service is available. Upgrading takes just seconds.”
++
+*Result:* The mocking service is switched off and then on. The format of the base URI that the mocking service generates now differs from the format of the previous base URI.
++
+Version 1 of the mocking service used base URIs that began with `https://mocksvc.mulesoft.com`.
++
+Version 2 uses base URIs that begin with `https://anypoint.mulesoft.com/mocking/api/v1/links/`.
+
+. If any third-party tools are testing endpoints in the API specification for that project, replace the previous base URI with the new base URI in the links that those tools are using.
+
+== New Features in Version 2 That You Can Use in API Designer
+
+Version 2 of the mocking service, which was released on January 10, 2019, introduced the following list of features that you can use in API Designer. For a full list of features, see the https://anypoint.mulesoft.com/exchange/org.mule.examples/mocking-service-2-api/version/v1/pages/Getting%20Started/[Getting Started page for the Mocking Service 2 API]. (The link requires that you authenticate to Anypoint Platform.)
```

Comment:
This link points to an Exchange Portal that explains how to use the Mocking Service 2 API. Not sure if that page will help or not the users understand the benefit of migrating to version 2 of mocking service

Reply:
Removed.